deps.edn

```diff
@@ -901,15 +901,17 @@
   ;; https://github.com/borkdude/clj-kondo/
   :lint/clj-kondo
   {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2021.12.16"}}
-   :main-opts  ["-m" "clj-kondo.main" "--lint" "src"]}
+   :main-opts  ["-m" "clj-kondo.main" "--lint" "src" "test"]}
```

Comment:
This change does fit with the title of the PR.  Can you remove from the PR and put in another one (or I can change it in a separate commit).  Thanks

Reply:
Undone, feel free to re-do it separately